.github/workflows/ff-bleeding.yml

```diff
@@ -0,0 +1,39 @@
+name: Fast-forward bleeding
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+env:
+  FF_ISSUE_TITLE: "No-FF: bleeding is behind master"
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+  release-tag-verification:
+    name: Fast-forward bleeding
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Criterion
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure git
+        run: |
+          git config user.name "github-actions"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
This is fine, but I'm curious where it's actually used now that we've removed any git object-writing operation; does the github actions robot user get used by `gh issue create` maybe?

Reply:
I think `gh issue create` works without this, I left it here because we still have to create an unpublished local commit, because Meson refuses to create a source tarball from uncommitted changes.